ssc/Pos/Ssc/Worker.hs

```diff
@@ -87,6 +87,8 @@ onNewSlotSsc
     :: (SscMode ctx m)
     => (WorkerSpec m, OutSpecs)
 onNewSlotSsc = onNewSlotWorker defaultOnNewSlotParams mempty $ \slotId diffusion ->
+    -- TODO: [CSL-198] Catch and report non-critical errors, as
+    -- 'onNewSlot' no longer swallows all exception and rethrows them instead.
```

Comment:
This worker may call `sendSscCommitment`, `sendSscOpening`, etc. Let's catch exceptions from these functions and restart. Other exceptions shouldn't be caught.